package/kedro_viz/server.py

```diff
@@ -139,16 +140,18 @@ def run_server(
     )
     args = parser.parse_args()
 
-    project_path = (Path.cwd() / args.project_path).absolute()
+    project_path = (
+        Path(args.project_path).resolve() if args.project_path else Path.cwd()
+    )
     bootstrap_project(project_path)
 
     run_process_kwargs = {
-        "path": project_path,
+        "path": args.project_path,
         "target": run_server,
         "kwargs": {
             "host": args.host,
             "port": args.port,
-            "project_path": str(project_path),
+            "project_path": str(args.project_path),
```

Comment:
since we are removing `path = Path(project_path) if project_path else Path.cwd()` from run_server, we can just pass the resolved `project_path` from line 143 here